tests/integration_tests/test_sklearn.py

```diff
@@ -357,3 +358,36 @@ def test_optuna_search_convert_deprecated_distribution() -> None:
     )
 
     assert optuna_search.param_distributions == expected_param_dist
+
+
+def test_callbacks() -> None:
+    class DummyCallback:
+        def __init__(self) -> None:
+            self.n_calls = 0
+
+        def __call__(self, study: "optuna.study.Study", trial: "optuna.trial.FrozenTrial") -> None:
+            self.n_calls += 1
+
+    callback1 = DummyCallback()
+    callback2 = DummyCallback()
+
+    X, y = make_blobs(n_samples=10)
+    est = SGDClassifier(max_iter=5, tol=1e-03)
+    param_dist = {}  # type: ignore
```

Comment:
How about optimising a hyperparameter rather than not optimising any param?


```suggestion
    param_dist = {"alpha": distributions.FloatDistribution(1e-04, 1e03, log=True)}
```